lib/command/diff.php

```diff
@@ -131,11 +152,131 @@ private function createDiff(array $schema, array $tables, $allowEmpty)
         foreach ($schema as $tableName => $table) {
             $diff->dropTable($tableName);
         }
+    }
 
-        if ($diff->isEmpty() && !$allowEmpty) {
-            return null;
+    /**
+     * @param rex_sql_table[]       $tables
+     * @param rex_ydeploy_diff_file $diff
+     */
+    private function handleFixtures(array $tables, rex_ydeploy_diff_file $diff)
+    {
+        $fixtureTableNames = [
+            'action',
+            'markitup_profiles',
+            'media_manager_type',
+            'media_manager_type_effect',
+            'metainfo_field',
+            'metainfo_type',
+            'module',
+            'module_action',
+            'redactor2_profiles',
+            'template',
```

Comment:
Angenommen, wir ändern beide gleichzeitig etwas an einem Template (kann an ganz unterschiedlichen Stellen sein). Nun commite und pushe ich als erstes.
Dann commitest du und willst pushen. Geht nicht, da ein Commit von mir ist. Also willst du den bei dir reinmergen, oder rebasen.
Da wirst du dann einen Conflict bekommen. Und zwar nicht bei der Diff-Datei, denn da hast du ja eine andere, mit anderem Zeitstempel hinzugefügt, sondern bei der fixtures.yml.
In dieser fixtures-Datei steht der Template-Inhalt aber in einer Zeile (Jede Zelle in der DB wird zu einer Zeile in der yml-Datei). Es ist nun also deine Aufgabe, in dieser elend langen Zeile den Konflikt aufzulösen. Und anschließend müsstest du unter Umständen noch eine neue, dritte Diff-Datei erzeugen, mit der Template-Version die unsere beiden Änderungen enthält.

Dagegen haben die Developer-Dateien große Vorteile, da es ganz normale PHP-Dateien sind, die nicht in eine Zeile gequetscht wurden. Wenn wir also in ganz unterschiedlichen Bereichen in der Datei gearbeitet haben, gibt es gar kein Problem. Und ansonsten, im Falle eines Konflikts, wäre er viel leichter aufzulösen, da leichter erkennbar, was der andere denn eigentlich geändert hat.